test/include/psen_scan_v2/ros_barrier.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2021 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#ifndef PSEN_SCAN_V2_TEST_ROS_BARRIER_H
+#define PSEN_SCAN_V2_TEST_ROS_BARRIER_H
+
+#include <chrono>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include "psen_scan_v2_standalone/util/async_barrier.h"
+
+namespace psen_scan_v2_test
+{
+class RosBarrier : public psen_scan_v2_standalone::util::Barrier
```

Comment:
Please provide a few words and maybe a minimal example on this class.

Reply:
Done. How do you like it?

Reply:
We could, but it would be duplicated as long as it belongs to `psen_scan_v2` since we need it in the standalone, too: https://github.com/PilzDE/psen_scan_v2/blob/feature/ros2_port/standalone/test/include/psen_scan_v2_standalone/util/matchers_and_actions.h#L26